### DISCUSSION TITLE
Brand rules interactive site

### DISCUSSION THREAD

**Discussion by karenrobersn** [Bixal/rapid-response-team#114] [Pitches]
### Problem
_The raw idea, a use case, or something we’ve seen that motivates us to work on this_

It's difficult to communicate, use, and maintain our brand because the elements are in different places, in various levels of refinement, and not entirely accessible to the people who need to use them.

Some of the specific problems include:

- Our "brand book," the primary source of truth, is a large PDF document sitting somewhere in Google Drive. It contains helpful narrative and background, but is lacking in practical usage guidelines. Not everyone has access to the Google Drive so some can't find it or don't even try.
- Some details about the brand — like copy, colors, imagery — are not easily available for reuse.
- In some cases, there are inconsistencies, errors, or lack of information about the details.
- There is no clear way for people to ask questions or give feedback on brand elements and usage.
- There is not enough information about how to use brand elements in ways that follow accessibility guidelines.

### Appetite
_How much time we want to spend and how that constrains the solution_

We should have a releasable product by the end of the calendar year, which practically means by about December 17th.

### Solution
_The core elements we came up with, presented in a form that’s easy for people to immediately understand_

The basic idea is to create a publicly available website that will organize information about the brand in a clear, accessible, and useful way so various stakeholders can find what they need and get guidance on how to use it.

These are the primary audiences and use cases to consider:

- Designers who are working _on_ the brand itself (maintaining source files, creating templates, updating guidance, etc.)
- Designers who are working _with_ the brand, working with internal stakeholders to create unique expressions of the brand for various marketing and communications purposes.
- Non-designers who are working with the brand, primarily using templates and other provided collateral for various marketing and communications purposes that do not require a designer to be directly involved.

High-level mockup:

![image](https://user-images.githubusercontent.com/5177337/143040836-2c7bfb32-4ace-493f-b144-600c4d3b8da8.png)

### Rabbit Holes
_Details about the solution worth calling out to avoid problems_

#### Authenticated access to files
Although the site will be publicly accessible, some of the files we link to will be on authenticated sites, like Sharepoint or Google Drive. Permission settings for Sharepoint can be confusing and not everyone has access to Google Drive. So we may have to dig into this or consider how we provide clear information about what to do if someone can't access something they need.

#### Collaboration with other practices
Although the Design Team has a lot of engagement with the brand, we are not the sole keepers of the brand. There are other stakeholders that will need to be involved, or at least have some awareness, of what we're doing. This includes Comms, Senior Leadership, and others. We should probably set up a RACI chart to get aligned on this.

### No-Gos
_Anything specifically excluded from the concept: functionality or use cases we intentionally aren’t covering to fit the appetite or make the problem tractable_

- We don't need to recreate the whole brand book. We should focus instead on surfacing the most practical information to make it easier for everyone at Bixal to represent the brand well.

**Comment by pglevy** [comment]
Hey @Bixal/rapid-response, I started building out the pitch with the Idea discussion Karen started and moved it to Pitches. This is first pass so feel free to ask questions or add comments and we can refine further before digging into the work.

**Reply by karenrobersn** [reply]
One thing I have observed based on this pitch is that you are envisioning a more robust brand site that includes most, if not all, of the features of the brand guide itself. In my own mind I was imagining something simple and more top level. Here are the colors, don't turn the monogram into a chicken etc etc. I only want to make sure we have the right people looking through the guide and discussing any changes we would be making to improve it, or the things we will be leaving alone. Its a lot of information and I am still learning exactly how it is implemented (or not) at Bixal.
